.rubocop.yml

```diff
@@ -403,6 +403,9 @@ Minitest/SkipEnsure:
 Minitest/UnreachableAssertion:
   Enabled: true
 
+Minitest/AssertEqual:
```

Comment:
Would you move the Minitest/AssertEqual cop above Minitest/AssertNil to keep cops in alphabetical order?